ios/HaapiReactNativeExample.xcodeproj/project.pbxproj

```diff
@@ -309,8 +311,9 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = HaapiReactNativeExample/HaapiReactNativeExample.entitlements;
+				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = ALY8Q7BNK9;
```

Comment:
Remove any references to us and the customer needs to configure Xcode Apple IDs etc (for passkeys logins)